broadcast/src/linked/parsed.rs

```diff
@@ -25,12 +25,12 @@ pub enum Error {
     InvalidPartial,
     #[error("Invalid threshold")]
     InvalidThreshold,
-    #[error("Sequencer parse error: {0}")]
-    Sequencer(String),
-    #[error("Payload parse error: {0}")]
-    Payload(String),
-    #[error("Signature parse error: {0}")]
-    Signature(String),
+    #[error("Invalid sequencer")]
+    InvalidSequencer,
```

Comment:
There is no good way to put a type in here given that we use Array traits everywhere (inner errors may be defined outside of the commonware library).